index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+ 
+
+    <header>
+
+        <nav class="nav-bar">
+      <h3 class="logo">My Logo</h3>
+      <img src="hamburger.svg" alt="hamburger-menu" class="nav-menu">
+        </nav>
+
+
+    </header>
+
+      <section class="intro">
+     <h2 class="yoda">I'm Yoda. <br> Glad to see <br> You! </h2>
+     <p class="intro-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+     <p class="connect">LET'S CONNECT
+     </p>
+     <img src="socialmediaicons.svg" alt="">
+    </section>
```

Comment:
- [ ] Lastly, at the bottom of your design, I noticed that you're missing the curved border bottom effect as seen in the Figma design. Your design needs to look exactly like the design so kindly implement this in your design as well :wink: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/anshuman7negi/my-portfolio/assets/102699958/f6d1d26b-d19e-46b2-b569-31d18a620fec) | ![image](https://github.com/anshuman7negi/my-portfolio/assets/102699958/ac35333a-82f2-4569-995e-ed38e5fcf027) |